### DISCUSSION TITLE
alt + v doesn't work (pastefrom Clipboard) and ...

### DISCUSSION THREAD

**Discussion by r-xiumu** [wezterm/wezterm#2301] [General]
This is my conf,  but "alt + v"  doesn't work. 
And  when I only press "CTRL + W"  instead of "CTRL + SHIFT + W" , close tab worked. 
Why?
```
	{
		key = 'V',
		mods = 'ALT',
		action = wezterm.action.PasteFrom 'PrimarySelection'
	},
	{
		key = 'W',
		mods = 'CTRL|SHIFT',
		action = wezterm.action.CloseCurrentTab{confirm=false}
	},
```

**Comment by MuhammedZakir** [comment]
> This is my conf, but "alt + v" doesn't work.

If you set `debug_key_events` to `true`, WezTerm will print received events, and if any action is triggered, it prints that too.

https://wezfurlong.org/wezterm/config/lua/config/debug_key_events.html

> And when I only press "CTRL + W" instead of "CTRL + SHIFT + W" , close tab worked.

`Ctrl`+`w` is by default mapped to `CloseCurrentTab` just like `Ctrl`+`Shift`+`w`. You can get a list of currently defined key assignments using `wezterm show-keys`.

**Reply by r-xiumu** [reply]
I resoved ,thanks!

**Comment by wez** [comment]
Looks like you wrote `V` (uppercase), so the key assignment will match `ALT+SHIFT+V` (uppercase `v`) but not `ALT+v` (lowercase `v`)

**Reply by r-xiumu** [reply]
Nice！“w” instead of 'W' worked. Thanks !
